### ISSUE TITLE
Create mesh networking prototype

### ISSUE THREAD

**Issue by threat** [maidsafe-archive/MaidSafe#233]
Mesh Networking implementations like Open Garden already exist. It would be great if data could not only be stored in a decentralized fashion but also networked in a decentralized fashion, particularly from one library in multiple programming languages.

**Comment by dirvine** [comment]
Absolutely, we are aware of some of these and do look particularly at the Open Source implementations. ZeroMq and those are also worth considering as related projects with implementations of mesh like networking.  The more these come together the better

**Comment by RoboMantis** [comment]
I'd love to hear more updates about the mesh networking!

**Comment by peidong-hu** [comment]
Yes. I too looking for the update.

**Comment by dirvine** [comment]
At the moment there are no updates on Mesh, a few visits and some talks, I was hoping to go to cccamp etc. next month, looks unlikely with work needing done here. CRUST is now multiprotocol aware so we have the door open.
